sycl/include/sycl/ext/intel/experimental/fpga_kernel_properties.hpp

```diff
@@ -84,6 +102,18 @@ inline constexpr pipelined_key::value_t<
     pipeline_directive_or_initiation_interval>
     pipelined;
 
+template <fpga_cluster_options_enum option =
+              fpga_cluster_options_enum::stall_free>
+inline constexpr fpga_cluster_key::value_t<option> fpga_cluster;
+
+inline constexpr fpga_cluster_key::value_t<
+    fpga_cluster_options_enum::stall_free>
+    use_stall_free_clusters;
+
+inline constexpr fpga_cluster_key::value_t<
+    fpga_cluster_options_enum::stall_enable>
+    use_stall_enable_clusters;
```

Comment:
This is probably more of a comment for the spec PR, but this is different from the current version of it`(use_stall_free_clusters` vs `stall_free_clusters`)

Reply:
got carried away during implementation when I saw that the current kernel attributes were use_stall_enable_clusters and use_stall_free_clusters. Removed "use_" from properties and tests.